### ISSUE TITLE
compressed GCI file not working with save exploit

### ISSUE THREAD

**Issue by kjuno** [redolution/gekkoboot#27]
I tried to use the `iplboot_xz.gci` according to this [instruction](https://github.com/redolution/iplboot/blob/master/README.md#gamecube-memory-card), but it does not work. It freezes when the savegame payload is executed. The uncompressed `iplboot.gci` file works fine.

**Comment by 9ary** [comment]
Compressed DOLs can be problematic sometimes. What version(s) of iplboot can you reproduce this with? And what games? I'm unable to test things on real hardware at the moment.

**Comment by kjuno** [comment]
precompiled iplboot r7 with Tom Clancy's Splinter Cell: Pandora Tomorrow

**Comment by kjuno** [comment]
At some day i tested something with picoboot and tried to use the compressed dol, but the entry point and load address was invalid, could that be the problem?
![what](https://user-images.githubusercontent.com/34090514/186500174-89f75021-8b9e-4462-89f4-bf4ca3a2e803.PNG)

**Comment by 9ary** [comment]
No, that's just an implementation detail for PicoBoot. I don't recall this being an issue for savegame exploits but I might be wrong, I'll investigate this later.

**Comment by kjuno** [comment]
ok weird, iplboot r6 works fine

**Comment by kjuno** [comment]
so entry point and load address isnt the case here

**Comment by 9ary** [comment]
Please try r8 as well in this case, that uses the same custom build of dolxz as r6. This should help narrow it down a little.

Side note, please avoid double-posting in quick succession like this. I get an email for each message you post. You can edit your messages instead.

**Comment by kjuno** [comment]
iplboot r8 works

**Comment by 9ary** [comment]
Hopefully this works with r9. @kjuno would you please test and close this issue if it works?

**Comment by kjuno** [comment]
sorry that it took over an year, works fine with gekkoboot r9.3, see Video below.

https://www.youtube.com/watch?v=f912RfUTB3w

**Comment by 9ary** [comment]
Thanks!
